src/apps/tezos/sign_tx.py

```diff
@@ -5,8 +5,13 @@
 from trezor.messages.TezosSignedTx import TezosSignedTx
 
 from apps.common import paths
-from apps.common.writers import write_bytes, write_uint8
+# from apps.common.writers import write_bytes, write_uint8
```

Comment:
it seems better to import these from `apps.common.writers` and only import `write_bool` from tezos?